### PR TITLE
FEAT: Added Cart page

### PR DESCRIPTION
**Pull request by kotesh-arya**
Screenshot from 2022-02-20 17-37-27

### CODE REVIEW

**Review by sks-21 — APPROVED** [review]
The code looks nice. Good to merge

File: Components/Cart/cart.html [hunk 1]
```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (khyatibaria) [inline Components/Cart/cart.html:7]: can add author meta tags

**Review by khyatibaria — COMMENTED** [review]
looks good to merge, just added a comment check

File: Components/Cart/cart.html (hunk 1 above)
Comment (kotesh-arya) [reply Components/Cart/cart.html:7]: Sure, will do that 👍🏻

**Review by ayush65 — APPROVED** [review]

**Pull request merged**